packages/xc-sidebar/src/Sidebar.tsx

```diff
@@ -4,44 +4,44 @@ import React, {
   ReactElement,
   useRef,
   useEffect,
-  ReactNode,
   useContext,
 } from 'react';
-import { ThemeContext } from 'styled-components';
-import { Scrollbars } from 'react-custom-scrollbars';
+import { ThemeContext, withTheme, ThemeProvider } from 'styled-components';
 import ResizeObserver from 'resize-observer-polyfill';
-import { IThemeNamespace } from '@xcritical/theme';
+
 import Arrow from './Arrow';
-import { sidebarTheme, ISidebarTheme } from './utils';
+import { ISidebarProps } from './interfaces';
 import {
+  Root,
   ResponsiveWrapper,
-  RightBorder,
-  CloseOpenButton,
-  RightBorderWrapper,
-  AntiSelect,
   ChildWrapper,
   SidebarWrapper,
   NavComponentWrapper,
+  SeparatorWrapper,
+  Separator,
+  CloseOpenButton,
+  AntiSelect,
+  Scrollbar,
 } from './styled/Sidebar';
 
 
-interface IWrapperProps {
-  navComponent?: ReactNode;
-  children?: ReactNode;
-  theme?: IThemeNamespace;
-  showScrollbar?: boolean | string;
-}
-
-export const Sidebar = ({
+export const PureSidebar = ({
   theme,
+  appearance = 'default',
```

Comment:
I think this component don't need appearance, but you already done it, so... okay)